quantum/quantum.c

```diff
@@ -646,6 +681,17 @@ bool process_record_quantum(keyrecord_t *record) {
               }
             #endif
             break;
+          case MAGIC_TOGGLE_CTL_GUI:
+            keymap_config.swap_lctl_lgui = !keymap_config.swap_lctl_lgui;
```

Comment:
this could lead to odd behavior. What if lctl_lgui is true and rctl_rgui is false. Is that intended?

Reply:
Is the same as ALT-GUI.  But I suspect that no, it wasn't intended. 

However, I don't feel that this is something that needs to be addressed here.